src/webview2/mod.rs

```diff
@@ -592,12 +596,27 @@ impl InnerWebView {
         token,
       )?;
     }
+    let scripts = attributes.initialization_scripts.clone();
+
+    webview.add_ContentLoading(
+      &ContentLoadingEventHandler::create(Box::new(move |webview, _| {
+        let Some(webview) = webview else {
+          return Ok(());
+        };
+
+        for (script, _) in scripts.iter().filter(|(_, for_main)| *for_main) {
+          Self::execute_script(&webview, script.clone(), |_| ())?;
+        }
+
+        Ok(())
+      })),
+      token,
+    )?;
```

Comment:
This looks fine but actually it sometimes injects the script at the wrong timing, for example, currently Tauri's helloworld example will throw an error saying `window.__TAURI__` is `undefined` but works if you reload the page or use `AddScriptToExecuteOnDocumentCreated`

@amrbashir I'm not sure if we want to keep this or just don't support injecting to main page only on Windows

Reply:
I guess we should revert? I mean we always injected into all frames, so I don't think it should be harmful, or do we still need this @tweidinger

Reply:
We moved to selective injection due to security issues like https://github.com/tauri-apps/tauri/security/advisories/GHSA-57fm-592m-34r7 - windows was always a bit of a special case since it allows injection into same origin but afaik we did not want this Auto injection behavior.